Runtimes/Core/cmake/modules/DefaultSettings.cmake

```diff
@@ -68,6 +68,18 @@ elseif(WIN32)
   set(SwiftCore_THREADING_PACKAGE_default "WIN32")
   set(SwiftCore_ENABLE_PRESPECIALIZATION_default ON)
   set(SwiftCore_CONCURRENCY_GLOBAL_EXECUTOR_default "dispatch")
+
+  set(SwiftCore_ENABLE_COMMANDLINE_SUPPORT_default ON)
+  if(BUILD_SHARED_LIBS)
+    set(SwiftCore_ENABLE_LIBRARY_EVOLUTION_default ON)
+  else()
+    set(SwiftCore_ENABLE_LIBRARY_EVOLUTION_default OFF)
+  endif()
+  set(SwiftCore_ENABLE_VECTOR_TYPES_default ON)
+  set(SwiftCore_ENABLE_RUNTIME_FUNCTION_COUNTERS_default ON)
+  set(SwiftCore_ENABLE_FILESYSTEM_SUPPORT_default ON)
+  set(SwiftCore_INSTALL_NESTED_SUBDIR_default ON)
+  set(SwiftCore_OPTIMIZATION_REMARKS_default "bitstream")
```

Comment:
It might be time to start writing caches. This is useful for certain use-cases, but I don't think it's necessary in all configurations.

Reply:
I think that the number of options here is pretty small, but yeah, eventually, that would be good.